src/ServiceControl.DockerImages/ServiceControl.DockerImages.csproj

```diff
@@ -78,7 +74,13 @@
     <MakeDir Directories="$(DockerfilesFolder)" />
   </Target>
 
+
   <Target Name="ExpandDockerfileTemplates" AfterTargets="CleanGeneratedDockerfiles">
+
+    <PropertyGroup>
+      <DockerfilesFolder>$(OutputPath)/dockerfiles/</DockerfilesFolder>
```

Comment:
Why are you moving where the dockerfiles are being created? This isn't necessary for copying the deploy files and causes problems for the release workflow which needs to know where they dockerfiles are.

Reply:
No, this is tested and has correct behavior. I needed to move it here otherwise `$(OutputPath)` does not have a value as its resolved too early.

Reply:
I added a comment in 0ca696be24b9e305f7aca5213683b7e5659422a2 to explain why its there and moved it in 0a59b67a9 to the task that uses it first.

Please share if you know of an alternative. The only alternative I see is using `$(OutputPath)` in each task but that defeats the purpose of `DockerfilesFolder` a bit as each task is than doing its on path concatenation.

Reply:
@ramonsmits I think you missed the point of my question. We discussed copying the deploy content into the bin folder, but not the dockerfiles. 

The dockerfiles don't need to be in the same place, and still aren't with your changes. Moving where they are breaks the release workflow.

Reply:
@bording gotcha, well, I added a .gitignore in there which you didn't like that is why I moved that to the `bin` folder so that the .gitignore isn't needed.

Reverted that in 395cf836fcc070d48b6ad3603c2f248be0bd0c2f with a comment too that that location is used by release CI.